docs/SPEC.md

```diff
@@ -325,6 +325,15 @@ Interpolation example:
 
 [IMPL#1775](https://github.com/influxdata/flux/issues/1775) Interpolate arbitrary expressions in string literals
 
+String interpolation now supports all basic type arbitrary expressions  without requiring casting to a string. 
```

Comment:
nitpit, but can you remove the `now` wording? We want the SPEC to read in the present tense as people reading the SPEC won't necessarily know or remember the history of Flux. 

In the SPEC we list the type classes can you add Stringable to that list?

Reply:
On a related note, should I talk to docs team about the updated functionality of string interpolation?

Reply:
Yes, ping them in their channel with a link to this PR